pkg/labels/labels.go

```diff
@@ -86,9 +86,6 @@ const (
 	// LabelSourceK8s is a label imported from Kubernetes
 	LabelSourceK8s = "k8s"
 
-	// LabelSourceMesos is a label imported from Mesos
-	LabelSourceMesos = "mesos"
```

Comment:
I see a few additional references to `mesos`/`Mesos`:

- `Documentation/spelling_wordlist.txt`
- `plugins/cilium-cni/types/types.go` (not sure if we still need them)

Reply:
:+1: fixed up.

Reply:
Evidently the spelling wordlist case must stay there as long as we mention the deprecation in the upgrade guide, see the recent failure:

```
Please fix the following spelling mistakes:
* Documentation/operations/upgrade.rst:379: (Mesos) 

If the words are not misspelled, run:
Documentation/update-spelling_wordlist.sh Mesos 
```

Reply:
Argh! Sorry.